.github/workflows/build.yaml

```diff
@@ -182,10 +182,15 @@ jobs:
             -   name: Convert code_samples usages for comment
                 if: steps.list.outputs.CODE_SAMPLES_CHANGE != '' && steps.diff.outputs.CODE_SAMPLES_DIFF != '0'
                 run: |
-                    echo '# code_samples/ change report' >> code_samples_usage.diff.md
+                    echo '# code_samples/ change report' > code_samples_usage.diff.md
                     echo '' >> code_samples_usage.diff.md
                     php tools/code_samples/code_samples_usage_diff2html.php $HOME/code_samples_usage.diff >> code_samples_usage.diff.md
                     echo '<a href="${{ steps.artifact.outputs.artifact-url }}">Download colorized diff</a>' >> code_samples_usage.diff.md
+                    if [[ `wc -m < code_samples_usage.diff.md | xargs` -ge $((2**16)) ]]; then
```

Comment:
Truncation happens over 65536 characters, `...*[Comment body truncated]*` replace the end so the result is exactly 65536 characters.